airbyte-cdk/java/airbyte-cdk/dependencies/src/main/java/io/airbyte/commons/logging/LoggingHelper.java

```diff
@@ -41,9 +41,10 @@ public String getCode() {
 
   @VisibleForTesting
   public static final String RESET = "\u001B[0m";
+  public static final String PREPARE_COLOR_CHAR = "\u001b[m";
 
   public static String applyColor(final Color color, final String msg) {
-    return color.getCode() + msg + RESET;
+    return PREPARE_COLOR_CHAR + color.getCode() + msg + PREPARE_COLOR_CHAR + RESET;
```

Comment:
looks like for some editors, the current coloring is a bit broken, so this is issuing a reset character before the coloring.